src/components/App.js

```diff
@@ -0,0 +1,16 @@
+import React          from 'react';
+import BooksList      from "../containers/BooksList";
+import BooksForm      from "../containers/BooksForm";
+import Navbar         from "../components/Navbar"
+import                     '../styles/style.css';
+function App() {
```

Comment:
There is no problem to use `function` but it will be good if you use ES6 arrow function there. To make consisting styling across your code base. 🙂